documentation/content/en/books/handbook/wayland/_index.adoc

```diff
@@ -50,11 +50,11 @@ endif::[]
 
 [[wayland-synopsis]]
 == Wayland Synopsis
-Wayland is a new display server, but it differs from Xorg in several important ways.
-First, Wayland is only a protocol that acts as an intermediary between clients using a different mechanism which removes the dependency on an X server.
-Xorg includes both the X11 protocol, used to run remote displays and the X server will accept connections and display windows.
-Under Wayland, the compositor or window manager provides the display server instead of a traditional X server.
+Wayland is a communication protocol that can replace a display server such as Xorg. It differs from Xorg in several important ways.
 
+First, Wayland is only a protocol that acts as an intermediary between clients using a mechanism which removes the dependency on an X server.
+Xorg includes both the X11 protocol, used to run remote displays, and the X server, used to accept connections and display windows.
+Under Wayland, the compositor or window manager provides the display server instead of a traditional X server.
 Since Wayland is not an X server, traditional X screen connections will need to utilize other methods such as VNC or RDP for remote desktop management.
 Second, Wayland can manage composite communications between clients and a compositor as a separate entity which does not need to support the X protocols.
```

Comment:
Sorry if the following tone is not so good, but I think this is really important.

This whole introductory paragraph feels very uncomfortable for the synopsis to bust out in the cold blood right into a disambiguation between xorg and wayland. Like walking into a highly technical argument :/ Except it is wrong, and also the wrong style of the handbook as a whole, to assume that the reader knows what we're arguing about, because everywhere they're realistically coming from 1. isn't using Xorg and 2. is almost certainly abstracting whatever they're using away from their sight.

This is the introduction paragraph to installing a graphical environment with wayland, which has been around and working for long enough that young people who might be coming from linux might not even know about xorg. People coming from windows or mac certainly won't, and there's no reason to assume they won't be. The rest of the handbook, especially the older parts, just explain to people how things work without assuming they know about anything, and that's friendly and cool, like this community.

Reply:
i really like the suggestions, flow a little nicer than the previous version.

Reply:
They absolutely flow nicer than before.

My suggestion is just a suggestion, and it is an honor for me to be here.

That said, please look at the introductory paragraph of the synopsis of every preceding chapter.